packages/discovery-provider/src/api/v1/users.py

```diff
@@ -2538,11 +2538,12 @@ class FullPurchasesCount(Resource):
         params={"id": "A User ID"},
     )
     @full_ns.expect(purchases_and_sales_count_parser)
-    @auth_middleware(purchases_and_sales_count_parser, require_auth=True)
+    @auth_middleware(purchases_and_sales_count_parser)
```

Comment:
I don't think you need `auth_middleware` at all if you aren't going to consume `authed_user_id`. I would take it out and remove the second function argument to `get`